update lang version docs (#10825)
dotnet/docs: docs/csharp/language-reference/configure-language-version.md

---
title: Select the C# language version - C# Guide
description: Configure the compiler to perform syntax validation using a specific compiler version
ms.date: 02/28/2019
---

# Select the C# language version

The C# compiler determines a default language version based on your project's target framework or frameworks. When your project targets a preview framework that has a corresponding preview language version, the language version used is the preview language version. When your project doesn't target a preview framework, the language version used is the latest minor version.

> For example, during the preview period for .NET Core 3.0, any project that targets `netcoreapp3.0` or `netstandard2.1` (both in preview), the C# 8.0 language (in preview) will be used. Projects targeting any released version will use C# 7.3 (the latest released version). This behavior means that any project targeting .NET Framework will use latest (C# 7.3). 

Reviewer: There's one sentence that seems to be missing the verb. Here's a suggestion but there are other ways to fix this:

```suggestion
For example, during the preview period for .NET Core 3.0, any project that targets `netcoreapp3.0` or `netstandard2.1` (both in preview) will use the C# 8.0 language (also in preview). Projects targeting any released version will use C# 7.3 (the latest released version). This behavior means that any project targeting .NET Framework will use latest (C# 7.3). 
```
